syntheseus/search/node_evaluation/base.py

```diff
@@ -93,10 +93,25 @@ def __init__(
         self,
         return_log: bool,
         return_negated: bool,
+        normalize: bool = False,
         temperature: float = 1.0,
         clip_probability_min: float = 1e-10,
         clip_probability_max: float = 0.999,
     ) -> None:
+        """Initialized the evaluator.
+
+        Args:
+            return_log: Whether to return the logarithm of the probability instead of the
+                probability itself.
+            return_negated: Whether to multiply the result by `-1`.
+            normalize: Whether to renormalize the output to be a distribution (or logarithms of
+                probabilities corresponding to a valid distribution). This is mostly useful when
+                `temperature != 1.0`, as otherwise the inputs would typically be normalized.
```

Comment:
Not sure if I agree that they would "typically be normalized" since normalization in some models could either be over *all* reactions (but the model only returns topK), or the probability could be normalized over "whether or not the reaction should be included", i.e. not normalized w.r.t the other reactions.

Reply:
Fair enough; I can drop the part after the comma then.